app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -0,0 +1,25 @@
+// Major library config
```

Comment:
This is basically a recreation of `application.scss` in webpacker, but also resolves the awful split we had between sprockets and npm.